Src/StartMenu/StartMenuDLL/StartMenuDLL.rc

```diff
@@ -1162,25 +1162,34 @@ BEGIN
-    IDS_STRING7106          "Windows XP Luna skin\n\nA start menu similar to the one in Windows XP.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7107          "Windows 8 skin\n\nDefault skin to use for Windows 8.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7108          "Midnight skin\n\nSkin with dark background.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7109          "Metro skin\n\nSkin that uses the start screen colors.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
-    IDS_STRING7110          "Metallic skin\n\nA start menu skin with metallic look.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
+    IDS_STRING7104          "Full Glass skin\n\nTransparent menu with large or small icons.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7105          "Smoked Glass skin\n\nSimple transparent menu with dark background.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7106          "Windows XP Luna skin\n\nA start menu similar to the one in Windows XP.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7107          "Windows 8 skin\n\nDefault skin to use for Windows 8.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7108          "Midnight skin\n\nSkin with dark background.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7109          "Metro skin\n\nSkin that uses the start screen colors.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7110          "Metallic skin\n\nA start menu skin with metallic look.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A>  2009-2017, Ivo Beltchev" 2009-2017, Ivo Beltchev"
+    IDS_STRING7111          "Immersive skin\n\nA skin that uses immersive colors and modern visual elements in the style of Windows 10.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> © 2009-2017, Ivo Beltchev"
```

Comment:
Other strings use `(c)` instead of `©`.
Lets keep it consistent (and use the same as in other stings here.

Reply:
Agree